fn main() {
    println!("It works, unexpectedly!");
}
